pages/learn/develop/hardware.md

```diff
@@ -27,8 +27,9 @@ __Note:__ If you are not using one of the Docker base images recommended in our
 For more details on interacting with external hardware, check out these guides:
 - [GPIO][gpio]
 - [i2C and SPI][i2c-spi]
+- [USB](usb)
 
```

Comment:
@anujdeshpande The link will go 404 as `usb` isn't referenced anywhere in the link footer. Check out reference links used in the document here 
https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs and add a reference for `usb`

Reply:
@anujdeshpande ?